File: js/class.js
class Foo {
  counter = 0;

  constructor(value = '') {
    this.value = value;
  }

  getValueAndCounter() {
    return this.value + (++this.counter);
  }
}

const foo = new Foo('Hello');
console.log(foo.getValueAndCounter());
console.log(foo.getValueAndCounter());

const PI = 3.14;
PI += 1;
console.log(PI);

File: py/class.py
class Foo:
  counter = 0

  def __init__(self, value = ''):
    self.value = value

  def get_value_and_counter(self):
    self.counter += 1
    return self.value + str(self.counter)

foo = Foo('Hello')
print(foo.get_value_and_counter())
print(foo.get_value_and_counter())


PI = 3.14

PI += 1

print(PI)
